tomviz/Utilities.cxx

```diff
@@ -1130,4 +1131,40 @@ bool vtkRescaleControlPoints(std::vector<vtkTuple<double, 4>>& cntrlPoints,
   }
   return true;
 }
+
+bool getVoxelValue(vtkImageData* data, double point[3], int indices[3],
```

Comment:
Let's define API with vtkVector3d and vtkVector3i, why not return the value, and take an optional bool by reference for ok

Reply:
You can use .GetData() to get a pointer to the double* or int* for the vtkVector types